src/init/initializeQuiz.js

```diff
@@ -23,6 +23,9 @@ const setupQuizHTML = () => {
 
   quizContainer.appendChild(questionContainer);
 
+  const quizQuestions = createQuestionElement(quizData.questions);
+  [...quizQuestions.children].forEach((question) => questionContainer.appendChild(question));
```

Comment:
```suggestion
  quizQuestions.children.forEach((question) => questionContainer.appendChild(question));
```

I think you can just do this, right?

`[...[]] = []`

Reply:
Nope I think we need it because if I remove it I get the following erron: ```quizQuestions.children.forEach is not a function```